bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js

```diff
@@ -189,6 +189,7 @@ const uploadAndConvertPresentations = (
 
 const setPresentation = (presentationId, podId) => {
   makeCall('setPresentation', presentationId, podId);
+  Session.set('currentPresID', presentationId);
```

Comment:
Shouldn't we wait for confirmation from server side before setting the presenter's [local] current presentation?